bin/setup

```diff
@@ -7,4 +7,6 @@ if [ "${1-}" = "circleci" ]; then
     mix local.rebar --force
 fi
 
-mix do deps.get, compile --warnings-as-errors
+mix deps.get
+mix deps.compile
```

Comment:
any reasoning behind not having warning as errors?

Reply:
Yeah I had that originally, but there's a bunch of warnings in the libs we use, which cause the build to fail. I couldn't figure out a way to get it to apply _only_ to our code. I didn't want to spend too much time on it, so I punted for now.